go.mod

```diff
@@ -83,7 +83,7 @@ require (
 	github.com/cobaugh/osrelease v0.0.0-20181218015638-a93a0a55a249
 	github.com/containerd/cgroups v0.0.0-20210114181951-8a68de567b68
 	github.com/containerd/containerd v1.5.0-beta.4
-	github.com/containerd/typeurl v1.0.1
+	github.com/containerd/typeurl v1.0.2
```

Comment:
I don't understand why this is being updated as well in the same PR (AFAIK it's a completely unrelated dep)

Reply:
Let's try recreating the PR

Reply:
Fixed